apis/s3/src/main/java/org/jclouds/s3/options/ListBucketOptions.java

```diff
@@ -37,6 +37,15 @@
 public class ListBucketOptions extends BaseHttpRequestOptions implements Cloneable {
    public static final ListBucketOptions NONE = new ListBucketOptions();
 
+   public ListBucketOptions withListType(String listType) {
```

Comment:
Should this be `int` to match other getters and setters?

Reply:
It's a bit strange, the only value that's allowed is 2. That's how list objects v2 is triggered. 
With v1 you don't set list-type at all. So there's no list type v1. 
I can just make it an int and only allow value 2.